pages/cloudflare/index.mdx

```diff
@@ -52,9 +52,9 @@ We will update the list as we progress towards releasing 1.0.
 - [x] [Middleware](https://nextjs.org/docs/app/building-your-application/routing/middleware)
 - [x] [Image optimization](https://nextjs.org/docs/app/building-your-application/optimizing/images) (you can integrate Cloudflare Images with Next.js by following [this guide](https://developers.cloudflare.com/images/transform-images/integrate-with-frameworks/))
 - [x] [Partial Prerendering (PPR)](https://nextjs.org/docs/app/building-your-application/rendering/partial-prerendering)
+- [x] [Incremental Static Regeneration (ISR)](https://nextjs.org/docs/app/building-your-application/data-fetching/incremental-static-regeneration)
```

Comment:
We should say that only the `direct` mode is support for now and it is not suitable for production